index.android.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { AppRegistry } from 'react-native';
+import { Router, Scene } from 'react-native-router-flux';
```

Comment:
I do not like `router` concept in native applications at all, since we already have scenes, but it's OK for now.